snippets/csharp/programming-guide/discards/discard-pattern2.cs

```diff
@@ -21,7 +21,7 @@ private static void ProvidesFormatInfo(object obj)
          Console.Write("A null object reference: ");
          Console.WriteLine("Its use could result in a NullReferenceException");
       }
-      else if (obj is var _)
+      else
```

Comment:
Let's leave this sample as is for the moment.  Issue dotnet/docs#8727 points to the need for a  better sample. I'd rather leave this in place until we make a new sample that shows the usage better.